homeassistant/components/livisi/climate.py

```diff
@@ -145,10 +149,25 @@ async def async_added_to_hass(self) -> None:
     def set_hvac_mode(self, hvac_mode: HVACMode) -> None:
         """Do nothing as LIVISI devices do not support changing the hvac mode."""
 
+    def _update_hvac_state(self) -> None:
```

Comment:
Why not implement the hvac_action property instead?

Reply:
Sure. I didn't know this was the preferred way..